source/find_getmore_killcursors_commands.rst

```diff
@@ -745,6 +540,8 @@ More in depth information about passing read preferences to Mongos can be found
 
 Changes
 =======
+2022-01-?? Add comment option.
```

Comment:
Depending on whether my suggestions above are applied, this may be changed to "replace examples/tables for find, getMore, and killCursors with server manual links"

Reply:
Updated.